### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by rbreva**
### Blog app - Add forms

-Method named current_user, created using the first user.
-Forms for Post and Comments created
-Like's buttom created.

Thanks for the review

Mauricio Mendoza

### CODE REVIEW

File: app/views/posts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<section class="users_section">
+  <h1>New post</h1>
+  <%= form_with model:Post do |f| %>
+    <%= f.label :title %><br>
+    <%= f.text_field :title, placeholder: 'Title'%>
+    <%= f.label :text %><br>
+    <%= f.text_area :text, placeholder: 'Text' %>
+    <%= f.submit "Create Post"%>
+  <% end %>
+</section>
```
Comment (muneebulrehman) [inline app/views/posts/new.html.erb:10]: Though you have a view for creating a new post, I was unable to locate a link on the UI to perform the same action. Kindly add a link for creating a new post for the current user. ⬇️ As you can see there is no link for it.
image
image

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,17 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show] do
```
Comment (muneebulrehman) [inline config/routes.rb:5]: There should be a route for creating a new post that is missing here. Kindly add a routes for the same.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @rbreva , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
Hi @rbreva 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the project requirement 👍🏽
- Descriptive README 📖
- No linter error ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**